src/buildHTML.js

```diff
@@ -1215,6 +1220,18 @@ groupTypes.leftright = function(group, options) {
     // Add it to the beginning of the expression
     inner.unshift(leftDelim);
 
+    // Handle middle delimiters
+    if (hadMiddle) {
+        for (i = 1; i < inner.length; i++) {
+            if (inner[i].isMiddle) {
+                // Apply the options that were active when \middle was called
+                inner[i] = delimiter.leftRightDelim(
```

Comment:
The documentation for `leftRightDelim` should be updated, but let's leave this a TODO for now.